recipes/datalad-container/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "datalad-container" %}
+{% set pypiname = "datalad_container" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypiname[0] }}/{{ pypiname }}/{{ pypiname }}-{{ version }}.tar.gz
+  sha256: 3e9c87e1aa712455863c8755fc480413297ddd820086312753123042fed8c139
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<32]
```

Comment:
Can this be noarch?
```suggestion
  noarch: python
```

Reply:
hm... in principle it is indeed a noarch package... Greedy me then would need to part with unittesting across platforms which, despite the package being a noarch, could unravel a variety of issues.  And ATM only on linux we have majority of  functionality available since that is where `git-annex` package is provided.  If we go `noarch` and build happens on non-linux, pretty much no useful tests would be ran to verify proper operation.

Reply:
That okay as noarch packages are only built on Linux in feedstocks. Pull requests in staged-recipes with failing Windows/macOS CI are still merged if the reason is known.

Reply:
ok, let's then try to proceed that way.  I will accept your suggestions now. Thank you!